cerberus-audit-logger-athena/src/main/java/com/nike/cerberus/audit/logger/listener/AthenaLoggingEventListener.java

```diff
@@ -71,30 +71,20 @@ public void onApplicationEvent(AuditableEvent event) {
                       .put(
                           "principal_type",
                           cerberusPrincipal
-                              .map(p -> cerberusPrincipal.get().getPrincipalType().getName())
+                              .map(p -> p.getPrincipalType().getName())
                               .orElse(AuditableEventContext.UNKNOWN))
                       .put(
                           "principal_token_created",
                           cerberusPrincipal
-                              .map(
-                                  p ->
-                                      cerberusPrincipal
-                                          .get()
-                                          .getCreated()
-                                          .format(ATHENA_DATE_FORMATTER))
+                              .map(p -> p.getCreated().format(ATHENA_DATE_FORMATTER))
```

Comment:
These look more like fixes, are we sure they are equivalent?

Reply:
yes they are equivalent